src/debug.c

```diff
@@ -73,6 +74,14 @@ void printCrashReport(void);
 void bugReportEnd(int killViaSignal, int sig);
 void logStackTrace(void *eip, int uplevel);
 
+/** This test creates 2 additional threads and calls ThreadsManager_runOnThreads
+ * with a callback that returns a string from each thread. Then writes the outputs to the log file.
+ * This test is called by sending "debug get-threads-stacktraces" command.
+ * NOTE: if the system is not linux, it does nothing.
+ * TODO: Once we implement getting the process's threads backtraces, replace dummy string with the actual backtraces.
+*/
+void ThreadsManager_test(void);
```

Comment:
this whole test can be replaced by simply sending using DEBUG SEGFAULT and watching the stack trace of other threads (like bio.c)